test/org/javarosa/model/xform/SmsSerializingVisitorTest.java

```diff
@@ -10,123 +10,110 @@
 
 import java.io.File;
 import java.io.IOException;
-import java.util.ArrayList;
+import java.util.Arrays;
 import java.util.List;
 import java.util.regex.Matcher;
 import java.util.regex.Pattern;
 
-import static org.javarosa.test.utils.Utils.convertToString;
 import static org.junit.Assert.assertFalse;
-import static org.junit.Assert.assertTrue;
 import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
 
 /**
  * Performs a comprehensive suite of tests on the results of a form
  * being serialized to the SMS format.
  */
-public class SMSSerializingVisitorTest {
-    private FormParseInit formParser;
-    private String SMS;
+public class SmsSerializingVisitorTest {
+    private String sms;
     private static final String delimiter = " ";
-    private FormEntryController formEntryController;
-    private FormInstance formInstance;
 
     @Before
     public void setup() throws IOException {
-        formParser = new FormParseInit();
+        FormParseInit formParser = new FormParseInit();
         formParser.setFormToParse(new File(PathConst.getTestResourcePath(), "sms_form.xml").toString());
-        formEntryController = formParser.getFormEntryController();
-        formInstance = formEntryController.getModel().getForm().getInstance();
+        FormEntryController formEntryController = formParser.getFormEntryController();
```

Comment:
We have a new helper method to find form files, and a new convention where we place the form ... well, see this commit comment:
https://github.com/dcbriccetti/javarosa/commit/272ca8cd5b09659ea8c8fc7dbf36dd81c3ebaefa